static/app/views/automations/hooks/utils.tsx

```diff
@@ -47,3 +48,9 @@ export function useConnectedIds({storageKey, initialIds = []}: UseConnectedIdsPr
 }
 
 export const NEW_AUTOMATION_CONNECTED_IDS_KEY = 'new-automation-connected-ids';
+export function useAutomationProjectIds(automation: Automation): string[] {
```

Comment:
making x requests to get project ids seems kinda weird. Are we using this anywhere?

Reply:
We need to get connected projects for the automation list view

<img width="942" alt="Screenshot 2025-05-15 at 3 27 50 PM" src="https://github.com/user-attachments/assets/8cfee8a7-c351-4e0e-bf45-bc4581e91291" />

Reply:
shouldn't the project ids exist in the list returned from that api

Reply:
yeah we've had a lot of back-and-forth discussion on this data model... it is weird! automations aren't associated directly with projects but only to detectors which are then associated with projects. so there's a lot of hoop jumping just to display these project badges, which would also be surfaced in the connected monitors tooltip.